### ISSUE TITLE
Is it possible to add a way to disable the web server on port 80

### ISSUE THREAD

**Issue by MichelRabozee** [kadrim/proxy4plex#17]
Hi,

I run the proxy Plex on a MacMini. The MacMini already runs a web server (on port 80).

 The issue is that in OS X, there is no way to modify the load order of the daemons (first the daemons in /Library/LaunchDaemons, then the daemons in the /System/Library/LaunchDaemons, and as the proxyplex is in the first directory (where the user may put global system items) and the web server launch item in the System directory (which is not modifiable by the user), the proxyplex starting a web server on port 80 prevents the global web server to start afterwards.

Hence is it possible to have a way to disable the "http.ListenAndServe(":80", nil)" as appearing in (personally I do not need the side loading system, and having it always running (and blocking other uses of port 80) whilst this functionality is used only once for app installation seems overkill):
```
 	go func() {
		log.Println("Trying to start app-deployer on port 80 ...")
		http.ListenAndServe(":80", nil)
	}()
```

Hoping to be heard,

Best regards

**Comment by kadrim** [comment]
good point, i will implement an argument to disable the sideloading (hence disabling the use of port 80)

**Comment by kadrim** [comment]
implemented in release 0.6.0: https://github.com/kadrim/proxy4plex/releases/tag/0.6.0
